implicit.cabal

```diff
@@ -43,6 +43,7 @@ Library
         -rtsopts
         -funfolding-use-threshold=16 
         -fspec-constr-count=10
+        -fno-warn-tabs
```

Comment:
Drive-by review: Has `-fno-warn-tabs` been added because there are tabs in the current code?

It might be useful to remove them as well, as part of a 7.10 patch.